eta/core/image.py

```diff
@@ -458,6 +448,70 @@ def from_dict(cls, d):
             filename=filename, metadata=metadata, attrs=attrs, objects=objects)
 
 
+class ImageSetLabelsDirectory(DirectoryContainer):
+    '''ImageSetLabelsDirectory is a DirectoryContainer that represents a list
+    of ImageLabels on disk.
+
+    See eta.core.serial.DirectoryContainer for more information and please
+    USE WITH CARE.
+    '''
+
+    _ELE_CLS = ImageLabels
+
+    _ELE_CLS_FIELD = "LABELS_CLS"
+
+    _ELE_ATTR = "labels_map"
+
+    _ELE_DIR_ATTR = "labels_dir"
+
+
+class BigImageSetLabels(ImageSetLabels):
```

Comment:
Following up. That all makes sense to me! It would make everything much more manageable. Just don't know how much existing code assumes a simple `list` as the container in the `ImageSetLabels`/`VideoSetLabels`.

It seems worth the change probably, regardless.